src/common/Graylog.cc

```diff
@@ -24,7 +24,7 @@ Graylog::Graylog(const SubsystemMap * const s, std::string logger)
       m_log_dst_valid(false),
       m_hostname(""),
       m_fsid(""),
-      m_logger(logger),
+      m_logger(std::move(logger)),
```

Comment:
logger is not a rvalue *reference*. so this does not help with the performance. and neither do i think it's a hotspot here.